docs/guides/getting-started/setup/sveltekit.mdx

```diff
@@ -0,0 +1,218 @@
+---
+description: Build a Tauri app using SvelteKit as the frontend build tool
+pagination_next: guides/development/development-cycle
+sidebar_position: 3
+sidebar_custom_props:
+  doc_card_image: /img/guides/getting-started/setup/svelte.svg
+---
+
+import TauriInit from './_fragments/_tauri-init.mdx'
+import Commands from './_fragments/_commands.mdx'
+import Intro from './_fragments/_intro.mdx'
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import Command, { InstallTauriApi } from '@theme/Command'
+
+# SvelteKit
+
+This guide will walk you through creating your first Tauri app using the [SvelteKit] frontend framework.
+
+<Intro />
+
+Here's a preview of what we will be building:
+
+![Application Preview](./assets/sveltekit/sveltekit.png)
+
+## Create the Frontend
+
+[SvelteKit] is a Svelte Frontend which is primarily designed for Server-Side Rendering (SSG). To make SvelteKit work with Tauri we are going to disable SSR and use [`@sveltejs/adapter-static`] to create a frontend based on Static-Site Generation (SSG).
+
+SvelteKit comes with a scaffolding utility similar to [`create-tauri-app`] that can quickly setup a new project with a number of customization options. For this guide we will select the [TypeScript] template, with ESLint and Prettier enabled.
+
+<Tabs groupId="package-manager">
+  <TabItem value="npm">
+
+```shell
+npm create svelte@latest
+```
+
+  </TabItem>
+
+  <TabItem value="Yarn">
+
+```shell
+yarn create svelte
+```
+
+  </TabItem>
+
+  <TabItem value="pnpm">
+
+```shell
+pnpm create svelte
+```
+
+  </TabItem>
+</Tabs>
+
+1. _Project name_  
+   This will be the name of your JavaScript project. Corresponds to the name of the folder this utility will create but has otherwise no effect on your app. You can use any name you want here.
+
+2. _App template_  
+   We will select `Skeleton project` for a barebones template. If you want to play around with a more complete SvelteKit example you can select `SvelteKit demo app`.
+
+3. _Type checking_  
+   Whether you want type checking via JSDoc or TypeScript in your project. For this guide, we assume you choose TypeScript.
+
+4. _Code linting and formatting_  
+   Whether you want to start your project with ESLint for code linting and Prettier for code formatting. There won't be other mentions about this in this guide, but we recommend enabling these 2 options.
+
+5. _Browser testing_  
+   SvelteKit offers built-in Playwright support for browser testing. Since Tauri apis don't work in Playwright, we assume you disable this option. Check out our [WebDriver documentation] for alternatives using Selenium or WebdriverIO instead of Playwright.
```

Comment:
```suggestion
   SvelteKit offers built-in Playwright support for browser testing. Since Tauri APIs don't work in Playwright, we recommend not adding it. Check out our [WebDriver documentation] for alternatives using Selenium or WebdriverIO instead of Playwright.
```